json-path/src/main/java/com/jayway/jsonpath/internal/function/latebinding/ILateBindingValue.java

```diff
@@ -0,0 +1,18 @@
+package com.jayway.jsonpath.internal.function.latebinding;
+
+/**
+ * Obtain the late binding value at runtime rather than storing the value in the cache thus trashing the cache
+ *
+ * Created by mattg on 3/27/17.
```

Comment:
I don't really like these file creation comments. IMHO they add no information that you cannot obtain from Git and usually become more and more irrelevant as time passes and other people edit the file. That's just my two cents of course, if @kallestenflo doesn't mind, I won't either. ;-)